x-pack/plugins/security_solution/public/timelines/components/graph_overlay/index.tsx

```diff
@@ -178,10 +191,13 @@ const GraphOverlayComponent = ({
       </EuiFlexGroup>
 
       <EuiHorizontalRule margin="none" />
-      <StyledResolver
-        databaseDocumentID={graphEventId}
-        resolverComponentInstanceID={currentTimeline.id}
-      />
+      {graphEventId !== undefined && (
+        <StyledResolver
+          databaseDocumentID={graphEventId}
+          resolverComponentInstanceID={currentTimeline.id}
+          indices={indices}
```

Comment:
This will render resolver with the SIEM default indices then once the signals index is fetched, it will rerender Resolver. This causes Resolver to start a request with just the default indices then cancel that request and re-request with the full set of indices. 

@jonathan-buttner @XavierM Should we hold off rendering Resolver until the signals index has been fetched?

Reply:
@andrew-goldstein Also worked on this.

Reply:
I think it probably makes sense to wait for the response from the `signals` index api request 🤷